metabase/metabase
# Update driver logos

Simple visual update of driver logos for the setup flow.

How to test:
- Run a fresh Metabase instance
- During the setup process, you should be able to see updated icons on the database connection step
